resdk/resolwe.py

```diff
@@ -243,7 +243,7 @@ def _process_file_field(self, path):
 
     def run(self, slug=None, input={}, descriptor=None,  # pylint: disable=redefined-builtin
             descriptor_schema=None, collections=[],
-            data_name='', src=None, tools=None):
+            data_name='', src=None, tools=None, use_existing=False):
```

Comment:
@mstajdohar: If `use_existing` is set to `True` and matching object exists, all other parameters (i.e. `collections`, `descriptor`,..) are ignored. Should I add a check that makes sure that other parameters are not given in this case?